### PR TITLE
[NUI] Add Value Indicator of Slider

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- Value Indicator is a tooltip of slider.
- Add some properties for value indicator :
  IsValueShown, ValueIndicatorText, ValueIndicatorSize, and ValueIndicatorUrl

### API Changes ###
- N/A

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 1]
```diff
@@ -66,7 +66,9 @@ public class SliderSlidingFinishedEventArgs : EventArgs
     /// <since_tizen> 6 </since_tizen>
     public partial class Slider : Control
     {
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        #region Fields
+
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
```
Comment (huiyueun) [inline src/Tizen.NUI.Components/Controls/Slider.cs:71]: We may not open bindable properties. Please remove wrong comment.

File: src/Tizen.NUI.Components/Controls/Slider.cs [hunk 2]
```diff
@@ -66,7 +66,9 @@ public class SliderSlidingFinishedEventArgs : EventArgs
     /// <since_tizen> 6 </since_tizen>
     public partial class Slider : Control
     {
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        #region Fields
```
Comment (m-macias) [inline src/Tizen.NUI.Components/Controls/Slider.cs:69]: I am not sure what is our policy regarding the '#region' directive, but there is a rule: 
StyleCop SA1124: You should not use regions.

File: src/Tizen.NUI.Components/Controls/Slider.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs:71]: I've just changed the sentences from 'tizen_6.0' to 'later'. Then, can I leave just [EditorBrowsable] tags?

File: src/Tizen.NUI.Components/Controls/Slider.cs (hunk 2 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Slider.cs:69]: Hmm Okay. Thank you for your review.
I've just used the '#region' to divide sections in a little long code in Slider.cs.
I'll remove these regions now. :)

**Review by huiyueun — APPROVED** [review]
Looks good to me

**Pull request merged**